src/constants/habitGuideConstants.tsx

```diff
@@ -0,0 +1,62 @@
+		paragraph: `타스는 자신이 원하는 사람이 되는 <span class="bold">‘과정’</span>에 중점을 두고 있어요.\n\n습관을 통해 '무엇을 하고 싶은지’가 아니라, <span class="bold">'어떤 사람이 되고 싶은지’</span>에 대해 질문하게 만들죠.\n\n이 접근 방식은 행동의 변화보다 더 깊은 수준, 즉 정체성의 변화가 목적입니다.\n\n정체성을 바꾸는 것이, 단순히 행동을 바꾸려는 시도보다 훨씬 강력한 변화를 이끌어낼 수 있거든요!`,
+	},
+	{
+		index: "02",
+		title: "시스템과 목표는 무슨 차이인가요?",
+		paragraph: `목표는 우리가 도달하고자 하는 '결과'에 초점을 맞춥니다.\n\n반면, <span class="bold"> 시스템은 그 목표를 달성하기 위해 매일 어떤 '과정'을 거칠 것인지에 대해 이야기해요.</span>\n\n목표가 우리를 원하는 방향으로 몰아가는 동기라면, 시스템은 그 목표에 이르는 구체적인 단계들이죠.\n\n예를 들어, 책을 더 많이 읽고 싶다면, 매일 밤 잠자기 전 20분 동안 책 읽는 시간을 만드는 것이 시스템이에요.`,
+	},
+	{
+		index: "03",
+		title: "좋은 습관 만들기_명확하게",
+		paragraph: `습관을 세울 때 '매일 아침 7시에 집 근처 공원에서 조깅하기'처럼 구체적으로 계획해 보세요.\n\n'더 운동하기'와 같은 모호한 목표보다, 언제 어디서 무엇을 할지를 명확히 하는 것이 <span class="bold">실천으로 옮기기 쉬워질 거예요.</span>\n\n이렇게 명확하게 계획을 세우면, 실제로 그 일을 해내는 데 한 걸음 더 다가갈 수 있답니다😊`,
+	},
+	{
+		index: "04",
+		title: "좋은 습관 만들기_매력적이게",
+		paragraph: `습관을 더 매력적으로 만들어보세요. 우리는 좋아하는 활동에 자연스레 더 많은 시간을 할애해요.\n\n예를 들어, 운동 후에 좋아하는 음악을 듣거나, 좋아하는 카페에서 커피를 마시는 등,\n\n습관을 더욱 즐길 수 있는 방법을 찾아보세요. <span class="bold">습관을 지속하고 싶은 마음이 커질 거예요.</span>`,
+	},
+	{
+		index: "05",
+		title: "좋은 습관 만들기_쉽게",
+		paragraph: `가능한 쉽게 시작해보세요. 준비 과정을 최소화하고, 시작할 수 있는 가장 간단한 단계부터 하는 거예요.\n\n예를 들어, 운동하기 전날에 운동복을 미리 준비해두고, 운동하는 데 필요한 최소한의 준비만 하는 거예요.\n\n<span class="bold">작게 꾸준히 하는 것이 장기적인 습관 형성으로</span> 이어집니다.`,
+	},
+	{
+		index: "06",
+		title: "좋은 습관 만들기_만족스럽게",
+		paragraph: `작은 성공도 크게 축하해주세요. 습관을 실천할 때마다, 자신에게 긍정적인 피드백을 주는 것이 중요해요.\n\n이것은 우리가 다음 번에도 습관을 실천하고 싶게 만드는 원동력이 된답니다.\n\n<span class="bold">작은 보상도 충분한 동기부여</span>가 되니 기억해 주세요.💪`,
+	},
+	{
+		index: "07",
+		title: "왜 ‘매일’ 하라는 거죠?",
+		paragraph: `꾸준함이 성공으로 가는 열쇠거든요. 매일 조금씩이라도 습관을 실천하는 것은 그 습관을 우리 삶의 자연스러운 부분으로 만듭니다.\n\n연구에 따르면 습관이 만들어지기까지 100번의 반복이 필요하다고 해요. <span class="bold">반복이 곧 습관 형성의 핵심</span>이랍니다.`,
+	},
+	{
+		index: "08",
+		title: "완벽지 않아도 괜찮아요",
+		paragraph: `실패는 성공으로 가는 길의 일부니까요. 중요한 것은 그 실패에서 배우고 다시 일어서는 태도랍니다.\n\n완벽을 추구하기보다는, <span class="bold">매 순간 최선</span>을 다하고, 필요할 때는 <span class="bold">다시 시작하는 용기</span>를 가지세요. 버디들이 응원할 거예요🙌`,
+	},
+	{
+		index: "09",
+		title: "내게 맞는 수준 찾기",
+		paragraph: `너무 쉽지도, 너무 어렵지도 않은, 딱 좋은 난이도를 '골디락스 존'이라고 해요.\n\n<span class="bold">자신에게 딱 맞는 수준</span>을 찾는 것은 습관에 중요하답니다.\n\n너무 쉬우면 도전 의식을 일으키지 못하고, 너무 어려우면 쉽게 포기하게 되기 때문이에요.\n\n<span class="bold">자신만의 골디락스 존</span>을 찾아, 그 안에서 습관을 형성해보세요.`,
+	},
+	{
+		index: "10",
+		title: "습관도 레벨업이 필요해요",
+		paragraph: `습관을 실천하면서 점차 익숙해지기 시작하면, 그 다음 단계는 조금씩 난이도를 높이는 거예요.\n\n예를 들어, 운동 습관이라면 더 길게, 더 빠르게, 또는 더 힘든 운동으로 점차 단계를 올려보세요.\n\n이렇게 해서 조금씩 업그레이드하면, 성장을 직접 느낄 수 있고, <span class="bold">습관이 더욱 흥미로워질 거예요.</span>`,
+	},
+	{
+		index: "11",
+		title: "정체성이 와닿지 않아요",
+		paragraph: `'나는 운동을 좋아하는 사람이다' 같은 정체성이 처음에는 자신과 잘 맞지 않게 느껴질 수 있어요.\n\n걱정하지 마세요. 정체성은 하루아침에 형성되는 것이 아니니 자연스러운 일이에요.\n\n작은 행동을 반복하면서 점차 그 행동이 자연스러워지고, 나중에는 그 정체성이 자신과 잘 맞는다고 느낄거에요.\n\n<span class="bold">정체성은 우리의 행동을 통해 서서히 내면화</span>되는 것이니까요.`,
+	},
+	{
+		index: "12",
+		title: "시각화&공유로 부스터 달기",
```

Comment:
혹시 전달받은 가이드 문서에 & 앞뒤로 띄어쓰기가 없었나요?
디자인 시안에는 & 앞뒤로 띄어쓰기가 들어가있습니다
문서와 동일하다면 그대로 넘어가도 될 것 같아요
```suggestion
		title: "시각화 & 공유로 부스터 달기",
```

Reply:
이거는 문서그대로내용